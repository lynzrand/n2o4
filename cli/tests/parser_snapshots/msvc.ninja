# From: Ninja docs, "deps"
msvc_deps_prefix = Note: including file:
rule cc
  deps = msvc
  command = cl /showIncludes -c $in /Fo$out
