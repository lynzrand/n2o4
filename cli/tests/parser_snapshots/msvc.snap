Ok(
    NinjaFile {
        global_scope: {
            "msvc_deps_prefix": "Note: including file:",
        },
        rules: {
            "cc": Rule {
                vars: {
                    "deps": Expandable(
                        [
                            Regular(
                                "msvc",
                            ),
                        ],
                    ),
                    "command": Expandable(
                        [
                            Regular(
                                "cl /showIncludes -c ",
                            ),
                            Var(
                                "in",
                            ),
                            Regular(
                                " /Fo",
                            ),
                            Var(
                                "out",
                            ),
                        ],
                    ),
                },
            },
        },
        builds: [],
    },
)
